examples/passport-login/src/__tests__/acceptance/passport-login.acceptance.ts

```diff
@@ -202,9 +202,7 @@ describe('example-passport-login acceptance test', () => {
         const params = {
           username: 'testuser',
           password: 'xyz',
-          // eslint-disable-next-line @typescript-eslint/camelcase
```

Comment:
I was scratching my head about why this `eslint-disable-next-line` is no longer needed and found the following entry in the [release notes](https://github.com/typescript-eslint/typescript-eslint/releases/tag/v3.0.0):

> camelcase - replaced by [naming-convention](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md)

IIUC, your change is dropping all checks for naming conventions, which is IMO a big step backwards - we would have to point naming convention violations manually in pull request reviews.

Please find a way how to enable `naming-convention` rule in our default config and how to configure it to be as close to `camelcase` as feasible.

Reply:
Added `naming-convention` rules. We can iterate over the details.